Open Judge System/Web/OJS.Web/Areas/Contests/Controllers/ResultsController.cs

```diff
@@ -390,7 +391,7 @@ public ActionResult Stats(int contestId, bool official)
             {
                 foreach (var problem in contestResults.Problems)
                 {
-                    var maxResultForProblemCount = contestResults.Results
+                    var maxResultForProblemCount = contestResults.Results.ToList()
```

Comment:
For `Count` we don't have to `ToList` the IQueryable, just refactor the `.?` check.